src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[fromFormat.length - 1]);
+  const fromDateObj = { separator: fromFormat[fromFormat.length - 1] };
```

Comment:
you don't need separator here

also, you do the same above: `fromFormat[fromFormat.length - 1]`. At least if you save it somewhere (in a variable) - you should use it